modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/java/com/liferay/portal/search/tuning/rankings/web/internal/display/context/RankingPortletDisplayBuilder.java

```diff
@@ -300,10 +304,15 @@ protected Boolean isShowCreationMenu() {
 	private RankingEntryDisplayContext _buildDisplayContext(
 		SearchHit searchHit) {
 
+		Ranking ranking = _documentToRankingTranslator.translate(
+			searchHit.getDocument(), searchHit.getId());
+
 		RankingEntryDisplayContextBuilder rankingEntryDisplayContextBuilder =
 			new RankingEntryDisplayContextBuilder(
-				_documentToRankingTranslator.translate(
-					searchHit.getDocument(), searchHit.getId()));
+				ranking,
+				_sxpBlueprintTitleProvider.getSXPBlueprintTitle(
```

Comment:
might have to check for a NPE here if SxpBlueprintTitleProvider is optional

Reply:
also maybe change the signature of getSXPBlueprintTitle and pass these args

`_sxpBlueprintTitleProvider.getSXPBlueprintTitle(
themeDisplay.getCompanyId(), themeDisplay.getLocale(), ranking.getSXPBlueprintExternalReferenceCode()));`